src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable react/jsx-key */
 import { Meta, StoryObj } from "@storybook/react";
-import { Accordion } from "./index";
+import { Text } from "../..";
+
+import { Accordion } from ".";
```

Comment:
hmm but this is not how it's being consumed by the user. User will always use what's exported from the index and I think we should also render that here. It tests whether our imports works (including typings) and what exactly it renders